### ISSUE TITLE
Features in inlay style over-draw each other

### ISSUE THREAD

**Issue by westnordost** [tangrams/tangram#632]
So I understood the documentation of the `inlay` and `overlay` blend mode to be in concept like `opaque`, only that it allows alpha within the layer so that it can be interlaced into the scene.

![inlay](https://user-images.githubusercontent.com/4661658/35632203-5741aa62-06a6-11e8-9890-5ffe079804c9.png)

Now, the outcome is a bit different as expected (for extruded features):
1. The renderer does not respect any z-order for each feature. It looks like every building here is just drawn on top of each other, as if each building was transparent (which it isn't, it is a solid color)
2. Even within one building, this is the case

This is something that of course can be expected from the `multiply `(and `add`) blend mode, but I find it certainly unexpected from the `inlay `/ `overlay `blend mode.

It is even more apparent in this demo scene, where I changed the map background to turqoise:
[Here is a minimal demo scene](https://tangram.city/play/?scene=https%3A%2F%2Fgist.githubusercontent.com%2Fanonymous%2F488ea148992d3ca2171b2983735e1478%2Fraw%2Fde4ee38463e7214d641fc506bb06aca89a4878ee%2Fscene.yaml#19.25000/53.56218/9.95356)
You see that the buildings themselves do not appear transparent, but they appear transparent _to each other only_. (Compare with `multiply `and `opaque `blend mode.)

So, to recap: As a user, I would expect the blend mode `inlay` and `overlay` to draw all the features in the draw rule as opaque (to each other) and fiddle in this layer into the rest of the scene.

I did not check if the same behavior occurs for tangram-ES.

**Comment by hjanetzek** [comment]
Have you tried blend mode `translucent`? :) The behavior that you expect needs a two-pass rendering - `inlay` and `overlay` are still useful for 2D geometries.

**Comment by westnordost** [comment]
Ah I see, I did not know about `translucent`, Tangram play did not show it as an option. So it does exactly that (two-pass-rendering) then(?).
Perhaps the tangram documentation could be updated - is there a public repository for that?

**Comment by tallytalwar** [comment]
Here is the public repo for the docs @westnordost: https://github.com/tangrams/tangram-docs and its updated with `blend: translucent` (https://github.com/tangrams/tangram-docs/blob/fde623b7a90dfad170f921f4a5a08188ae1e9f98/pages/styles.md#blend)

**Comment by westnordost** [comment]
Cool, thanks for the hint!
